default/python/AI/InvasionAI.py

```diff
@@ -126,6 +140,10 @@ def get_invasion_fleets():
                     'Best troop design %s can not be produces in at planet with id: %s\d' % (col_design, build_choices)
                 )
             n_bases = math.ceil((p_troops + 1) / troops_per_ship)  # TODO: reconsider this +1 safety factor
+            if n_bases > 20:  # fall back to standard troopers if would need more than 20 base troopers
+                print "ruling out base invasion troopers at %s due to excessive number required." % planet.name
```

Comment:
Use ```% planet``` instead of ```% planet.name``` - it gives both ID and name which is useful for debugging.

Example output: ```P_65<Kithrup Asteroids I>```